reports/report.html

```diff
@@ -0,0 +1,2 @@
 startGame is not defined\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at HTMLButtonElement.onclick (http://localhost:8080/:18:62)&quot;,&quot;diff&quot;:null},&quot;uuid&quot;:&quot;021dc0e0-56f2-45c1-bd9b-8a6deaa6972f&quot;,&quot;parentUUID&quot;:&quot;d9834d59-2a2d-46fd-b585-32b1747711fd&quot;,&quot;isHook&quot;:false,&quot;skipped&quot;:false}],&quot;suites&quot;:[],&quot;passes&quot;:[],&quot;failures&quot;:[&quot;8c58069f-b9dd-47a3-9f3b-e0b873740405&quot;,&quot;e319b97d-004a-4f12-b5d5-f323f4db81d8&quot;,&quot;864940a8-a6c1-4bb7-8d57-ecdc36542db8&quot;,&quot;5573e5c0-9dc6-40d6-a345-59d3d8fc403b&quot;,&quot;e94d39bf-701b-4b47-bfa2-4037fa174072&quot;,&quot;fcff7242-66be-4dcb-b5ed-b693de4afcb1&quot;,&quot;021dc0e0-56f2-45c1-bd9b-8a6deaa6972f&quot;],&quot;pending&quot;:[],&quot;skipped&quot;:[],&quot;duration&quot;:3020,&quot;root&quot;:false,&quot;rootEmpty&quot;:false,&quot;_timeout&quot;:2000}],&quot;passes&quot;:[],&quot;failures&quot;:[],&quot;pending&quot;:[],&quot;skipped&quot;:[],&quot;duration&quot;:0,&quot;root&quot;:true,&quot;rootEmpty&quot;:true,&quot;_timeout&quot;:2000}],&quot;meta&quot;:{&quot;mocha&quot;:{&quot;version&quot;:&quot;7.0.1&quot;},&quot;mochawesome&quot;:{&quot;options&quot;:{&quot;quiet&quot;:false,&quot;reportFilename&quot;:&quot;mochawesome&quot;,&quot;saveHtml&quot;:false,&quot;saveJson&quot;:true,&quot;consoleReporter&quot;:&quot;spec&quot;,&quot;useInlineDiffs&quot;:false,&quot;code&quot;:true},&quot;version&quot;:&quot;6.2.1&quot;},&quot;marge&quot;:{&quot;options&quot;:{&quot;reportDir&quot;:&quot;raw_reports&quot;,&quot;overwrite&quot;:false,&quot;html&quot;:false,&quot;json&quot;:true},&quot;version&quot;:&quot;5.1.0&quot;}}}" data-config="{&quot;reportFilename&quot;:&quot;report&quot;,&quot;reportDir&quot;:&quot;C:\\Users\\User2\\MateTask\\JS\\js_2048_game\\reports&quot;,&quot;reportTitle&quot;:&quot;js_2048_game&quot;,&quot;reportPageTitle&quot;:&quot;Mochawesome Report&quot;,&quot;inline&quot;:false,&quot;inlineAssets&quot;:false,&quot;cdn&quot;:false,&quot;charts&quot;:false,&quot;enableCharts&quot;:false,&quot;code&quot;:true,&quot;enableCode&quot;:true,&quot;autoOpen&quot;:false,&quot;overwrite&quot;:true,&quot;timestamp&quot;:false,&quot;ts&quot;:false,&quot;showPassed&quot;:true,&quot;showFailed&quot;:true,&quot;showPending&quot;:true,&quot;showSkipped&quot;:false,&quot;showHooks&quot;:&quot;failed&quot;,&quot;saveJson&quot;:false,&quot;saveHtml&quot;:true,&quot;dev&quot;:false,&quot;assetsDir&quot;:&quot;C:\\Users\\User2\\MateTask\\JS\\js_2048_game\\reports\\assets&quot;,&quot;htmlFile&quot;:&quot;C:\\Users\\User2\\MateTask\\JS\\js_2048_game\\reports\\report.html&quot;}"><div id="report"></div><script src="assets\app.js"></script></body></html>
```

Comment:
The line is too long and has too many attributes in a single line. Each element should be on a new line, and attributes should be indented with 2 spaces.